KonyAccelerate/controllers/mobile/frmMoreController/frmMoreController.js

```diff
@@ -0,0 +1,73 @@
+define({ 
+
+ //Type your controller code here 
+  /** @function moreMenuAnimate
+   *  		    Slide in menu items one by one
+   */
+  animationDuration : 0.5,
+  menuAnimateSlideIn: function(){
+    var self = this;
+    let menuWidgets = this.view.flxMoreWrapper.widgets();
+    let delayThrashold = 0;
+    menuWidgets.forEach(function(currentMenu, index){
+      let currentItemIdex = index;
+      currentMenu.animate(
+        kony.ui.createAnimation({
+          0:{left:"-80%","stepConfig":{}},
+          100:{left:"10%","stepConfig":{}}}),
+        {delay:delayThrashold,fillMode:kony.anim.FILL_MODE_FORWARDS,duration:self.animationDuration},
+        {animationEnd: function() {}}
+      );
+      delayThrashold = delayThrashold + 0.2;
+    });
+  },
+  /** @function menuAnimateSlideOut
```

Comment:
Please change the function comments style. We are using the below style for the complete application.

 /**
     * @function getCoordinates
     * @descripiton This function is used to construct the coOrdinates Information
     * @param eventLocationData The complete information about the event
     * @private
     */

Reply:
Done